app/src/main/java/com/sun/makeupwindow/ui/search/SearchContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.sun.makeupwindow.ui.search
+
+import com.sun.makeupwindow.base.BasePresenter
+import com.sun.makeupwindow.base.BaseView
+import com.sun.makeupwindow.data.model.Product
+
+interface SearchContract {
+    interface View : BaseView {
+        fun showProductsByWordSearch(product: List<Product>)
```

Comment:
```suggestion
        fun showProducts(product: List<Product>)
```
cha nó là Search rồi, thì ko cần đặt tên lặp lại